pkg/cmd/node/node_list.go

```diff
@@ -20,7 +20,7 @@ func newCmdNodeList(ctx api.Context) *cobra.Command {
 	var jsonOutput bool
 	cmd := &cobra.Command{
 		Use:   "list",
-		Short: "Show all nodes in the cluster",
+		Short: "Show all nodes in the cluster.",
```

Comment:
In help menus we don't have a `.` at the end of the description
```suggestion
		Short: "Show all nodes in the cluster",
```

Reply:
True, got confused because we did that in the old help menus.